Gemfile

```diff
@@ -57,7 +52,5 @@ end
 
 group :health do
   gem 'blinkr', '~> 0.2.7'
-  #gem 'blinkr', github: 'pmuir/blinkr'
-  #gem 'blinkr', path: '../blinkr'
```

Comment:
Eh, not really. I wouldn't expect anyone else to use a development version
of blinkr

On Wednesday, August 5, 2015, Paul Robinson <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/redhat-developer/developers.redhat.com/pull/348#discussion_r36330808>
> :
>
> > @@ -57,7 +52,5 @@ end
> >
> >  group :health do
> >    gem 'blinkr', '~> 0.2.7'
> > -  #gem 'blinkr', github: 'pmuir/blinkr'
> > -  #gem 'blinkr', path: '../blinkr'
>
> Looks like these are useful too. @LightGuard
> <https://github.com/LightGuard>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/348/files#r36330808>
> .
>


-- 
Sent from Gmail Mobile